tests/tensor_search/test_bulk_search.py

```diff
@@ -184,6 +184,13 @@ def setUp(self):
         ]
         self.selected_device = 'cpu'
 
+        tensor_search.create_vector_index(self.config, "index_name_1")
```

Comment:
Assuming these tests have been passing, why did you add this code now? There are no other changes

Reply:
Because `tensor_search.run_vectorise_pipeline` now will raise an error if the index does not exist. Because it now calls `get_index_info()` while determining the `text_query_prefix`.